x-pack/test_serverless/api_integration/test_suites/observability/synthetics/synthetics_enablement.ts

```diff
@@ -42,6 +42,8 @@ export default function ({ getService }: FtrProviderContext) {
   };
 
   describe('SyntheticsEnablement', () => {
+    // failsOnMKI, see URL
+    this.tags(['failsOnMKI']);
```

Comment:
1. This needs to be a function again, otherwise `this.tags` wouldn't work
2. WOuld be good to include the actual issue URL in the comment: `https://github.com/elastic/kibana/issues/180108`

Reply:
ah dang!!!  :onit: